<launch>
	
    <group ns="iai_donbot">
        <node pkg="giskardpy" type="giskard.py" name="giskard" output="screen">
            <param name="config" value="DonbotUnreal"/>
        </node>
    
        <node pkg="giskardpy" type="interactive_marker.py" name="giskard_interactive_marker" output="screen">
            <rosparam param="enable_self_collision">False</rosparam>
            <rosparam param="interactive_marker_chains">
                - [odom, gripper_tool_frame]
            </rosparam>
        </node>
    </group>

</launch>